docs/network-flow-visibility.md

```diff
@@ -610,3 +612,48 @@ and other Theia features, please refer to the
 **Starting with Antrea v1.7, support for the ELK Flow Collector has been removed.**
 Please consider using the [Grafana Flow Collector](#grafana-flow-collector-migrated)
 instead, which is actively maintained.
+
+## L7 Visibility
+
+In Addition to Layer 4 network visibility, Antrea adds Layer 7 network flow visibility
+enabling users to collect and visualize application-layer flow data in Theia.
+
+### Usage
+
+To achieve L7 (Layer 7) network flow visibility, the `L7Visibility` and `TrafficControl`
```

Comment:
Your new FeatureGate must be added to `docs/feature-gates.md` as well for documentation purposes

Reply:
Thanks! added to the doc